### PR TITLE
Add CERNBox project and proposals

### PR DESCRIPTION
**Pull request by labkode**

### CODE REVIEW

File: _gsocproposals/2018/proposal_CERNBoxNewUI.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+## Expected results
+Working web application that can manage the files stored in CERN EOS
+using the ownCloud Synchronisation Protocol based on WebDAV.
+
+## Requirements
+Javascript, HTML, CSS3, Vue.js, Golang
+
+## Mentors
+  * [Hugo Labrador](mailto:hugo.gonzalez.labrador@cern.ch)
+
```
Comment (etejedor) [inline _gsocproposals/2018/proposal_CERNBoxNewUI.md]: Every proposals needs at least two mentors, could you specify another one?

**Comment by labkode** [comment]
@etejedor I've updated the mentor list with two new mentors :)

File: _gsocproposals/2018/proposal_CERNBoxNewUI.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: Petabyte-Scale Cloud Storage File Manager
+layout: gsoc_proposal
+project: CERNBox
+year: 2018
+organization: CERN, AARNet, ownCloud
```
Comment (hegner) [inline _gsocproposals/2018/proposal_CERNBoxNewUI.md]: please add ownCloud and AARNet as "_gsocorgs/"

Comment (etejedor) [reply _gsocproposals/2018/proposal_CERNBoxNewUI.md]: Also, please specify organizations with the list syntax:
http://hepsoftwarefoundation.org/gsoc/guideline.html

**Comment by labkode** [comment]
@hegner I've added the missing organizations
Thx

**Pull request merged**